logging/log_collection_forwarding/log-forwarding.adoc

```diff
@@ -177,8 +177,8 @@ spec:
       outputRefs:
         - cw <11>
```

Comment:
serviceAccountName: clf-collector is need when CLF is not deployed as instance in openshift-logging

Reply:
<your_role_name> <8>   should be <your_secret_name>,  the content can include role_arn or key_id/key.

Reply:
>the content can include role_arn or key_id/key

This seems like it should be mentioned in the section about creating secrets rather than called out here, where the user is just being asked for the secret name, not to create the secret?